### DISCUSSION TITLE
my view of freedom

### DISCUSSION THREAD

**Discussion by cauerego** [libgit2/libgit2#6102] [General]
i noticed you have a modified GPL v2 license, to become compatible with apache it whatever.

and i work with my own research to find the best offer for my own usage in other projects (namely, achoz).

just curious to better understand: why you think gpl2 with this modification works so much better than all others?

did you try gpl 2.2 or 3 before, and got burned to get your work perpetually free?

or do you just want to reach the biggest audience possible, in exchange for losing some freedom?

because, to me, fsf clearly recommends agpl as the ideal for promoting as much freedom as their legal understanding goes today. and since i still can't really understand mostly anything when i read either license directly, i have to take real world instances.

**Comment by ethomson** [comment]
I'm afraid that you are unlikely to get a satisfactory answer.  There is nobody still working on this project who chose this license.

**Reply by cauerego** [reply]
thanks. which license they choose?

**Comment by axkibe** [comment]
I wondered similar, why not simply use LGPL?
But it's no use discussing, it's practically unchangeable now anyway (all authors would have to agree)
